addons/sale/models/sale_order_line.py

```diff
@@ -216,15 +217,12 @@ class SaleOrderLine(models.Model):
         compute='_compute_qty_to_invoice',
         digits='Product Unit of Measure',
         store=True)
-
-    analytic_tag_ids = fields.Many2many(
-        comodel_name='account.analytic.tag',
-        string="Analytic Tags",
-        compute='_compute_analytic_tag_ids', store=True, readonly=False,
-        domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
     analytic_line_ids = fields.One2many(
         comodel_name='account.analytic.line', inverse_name='so_line',
         string="Analytic lines")
+    analytic_distribution_stored_char = fields.Char(
+        compute="_compute_analytic_distribution_stored_char",
+        store=True)
```

Comment:
Instead of doing this in every module, you could do it once in the mixin and just inherit `_compute_analytic_distribution_stored_char` in the business models